SS2-Project/Assets/Starstorm2/Components/Pyro/PyroTornadoController.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Networking;
+using RoR2;
+using RoR2.Projectile;
+
+public class PyroTornadoController : NetworkBehaviour
+{
+    private List<CharacterBody> processedVictims = new List<CharacterBody>();
+    private CharacterBody ownerBody;
+    private ProjectileDamage projDmg;
+    private int burnStacks;
+    private Vector3 baseScale;
+
+    private float timer;
+    private float timeBetweenRefreshes = 0.5f;
+
+    public void Awake()
+    {
+        baseScale = transform.localScale;
+
+        if (TryGetComponent(out ProjectileController pc) && pc.TryGetComponent(out CharacterBody body))
+        {
+            ownerBody = body;
+        }
+
+        if (TryGetComponent(out ProjectileDamage dmg))
+        {
+            projDmg = dmg;
+        }
+    }
+
+    //public void FixedUpdate()
```

Comment:
delete if unused

Reply:
Doubt its that bad to have it there, it looks more to me like "I'd like this to happen but don't have the time to do so rn"